bpf/process/data_event.h

```diff
@@ -32,23 +36,31 @@ static long do_bytes(void *ctx, struct msg_data *msg, unsigned long arg,
 		     size_t bytes)
 {
 	size_t rd_bytes = 0;
-	int err, i;
+	int err, i __maybe_unused;
 
 #ifdef __LARGE_BPF_PROG
-#define __CNT 10
-#else
-#define __CNT 8
-#pragma unroll
-#endif
-	for (i = 0; i < __CNT; i++) {
+	for (i = 0; i < 10; i++) {
 		err = __do_bytes(ctx, msg, arg + rd_bytes, bytes - rd_bytes);
 		if (err < 0)
 			return err;
 		rd_bytes += err;
 		if (rd_bytes == bytes)
 			return 0;
 	}
-#undef __CNT
+#else
+#define BYTES_COPY                                                             \
+	err = __do_bytes(ctx, msg, arg + rd_bytes, bytes - rd_bytes);          \
+	if (err < 0)                                                           \
+		return err;                                                    \
+	rd_bytes += err;                                                       \
+	if (rd_bytes == bytes)                                                 \
+		return 0;
+
+#define BYTES_COPY_5 BYTES_COPY BYTES_COPY BYTES_COPY BYTES_COPY BYTES_COPY
+
+	BYTES_COPY_5
+	BYTES_COPY_5
+#endif
 
```

Comment:
nit: maybe undef BYTES_COPY and BYTES_COPY_5 here?

Reply:
yes, will remove it, thanks